tests/election.py

```diff
@@ -23,8 +24,27 @@
 @reqs.description("Stopping current primary and waiting for a new one to be elected")
 @reqs.can_kill_n_nodes(1)
 def test_kill_primary(network, args):
-    primary, _ = network.find_primary()
+    primary, backup = network.find_primary_and_any_backup()
     primary.stop()
+
+    # When the consensus is BFT there is no status message timer that triggers a new election.
```

Comment:
I am confused by this:

1. Didn't this test pass before? Which part of this PR changed that?
2. Shouldn't "X seconds without a signature from the primary" also trigger a view change? Why is a dummy request necessary?

Reply:
Previously primary info triggered a view change because it was going through the consensus now it is not so after killing the primary we need some request to go through the consensus, timeout, triggering a view change.

Reply:
Oh, I see.